packages/evolution-generator/src/scripts/labels_generator.py

```
@@ -252,6 +272,53 @@ def save_translations(
             print(f"An error occurred while saving translations to {file_path}: {e}")
             raise e
 
+    @classmethod
+    def __add_nested_translation(
```

Ruff (0.15.12)

[warning] Missing return type annotation for classmethod `__add_nested_translation`

Add return type annotation: `None`

(ANN206)